tests/e2e/config/env.setup.js

```diff
@@ -0,0 +1,5 @@
+global.process.env = {
+	...global.process.env,
+	// Gutenberg test util functions expect the test url to be at :8889, we change it to 8084.
```

Comment:
:question: 
Can we use `8889` in `config/default.json`, to stick to the defaults and remove this file, to reduce the boilerplate and number of config files?

Reply:
The port number `8084` is actually the default from WooCommerce and WC Admin. See the following links:

- https://github.com/woocommerce/woocommerce/tree/trunk/tests/e2e
- https://github.com/woocommerce/woocommerce/blob/f367fcd4f7eb39b14277e1bcb4083a7df66166e6/tests/e2e/env/utils/test-config.js#L49
- https://github.com/woocommerce/woocommerce-admin/blob/130eb95bae34a8efd83143c52cc08864efc15131/tests/e2e/config/env.setup.js#L4
- https://github.com/woocommerce/woocommerce-admin/blob/130eb95bae34a8efd83143c52cc08864efc15131/tests/e2e/config/default.json#L2

I just realized, since it is default, we don't really need this file. Removed in https://github.com/woocommerce/google-listings-and-ads/pull/801/commits/f9c9523d469f00cae04ea9233cbcac2fa095bfab.